Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -884,27 +884,27 @@ def negAt :
 variable {s}
 
 @[simp]
-theorem negAt_apply_of_isReal_and_mem (x : mixedSpace K) {w : {w // IsReal w}} (hw : w ∈ s) :
+theorem negAt_apply_ofIsReal_and_mem (x : mixedSpace K) {w : {w // IsReal w}} (hw : w ∈ s) :
```

Comment:
Sorry for missing the zulip conversation. My suggestion would be to drop the `of`
```suggestion
theorem negAt_apply_IsReal_and_mem (x : mixedSpace K) {w : {w // IsReal w}} (hw : w ∈ s) :
```
But it's only a mild gut feeling, not a strong opinion.
Have you considered this?

Anyway, I don't want to hold up the PR over this.

bors d+

Reply:
I can live with that (and it makes the name shorter :)). Thanks!

Reply:
@jcommelin, just to be sure, should it be `negAt_apply_IsReal_and_mem` or `negAt_apply_isReal_and_mem`?

Reply:
Naming convention says `isReal` so I'll go with that.

Reply:
But, then it's really similar to the names used in the other case.. So, in the end, I think I'll use `ofIsReal` and `ofIsComplex`.

Reply:
Well, in the end, I decided to go with Johan's suggestion but with lower camel case. Sorry for all the hesitation and the noise